SIGNALESP/cc1101.h

```diff
@@ -0,0 +1,522 @@
+// cc1101.h
+
+#ifndef _CC1101_h
+#define _CC1101_h
+
+#if defined(ARDUINO) && ARDUINO >= 100
+	#include "Arduino.h"
+#else
+	#include "WProgram.h"
+#endif
+#include <EEPROM.h>
+#include "output.h"
+
+#ifdef ESP8266
+	#include <SPI.h>
```

Comment:
anbei die Compiler-Fehler ohne SPI-library

In file included from SIGNALESP\SIGNALESP.ino:38:0:

sketch\cc1101.h: In function 'uint8_t cc1101::sendSPI(uint8_t)':

cc1101.h:223: error: 'SPDR' was not declared in this scope
   SPDR = val;                                      // transfer byte via SPI
   ^

cc1101.h:224: error: 'SPSR' was not declared in this scope
   while (!(SPSR & _BV(SPIF)));                     // wait until SPI operation is terminated
            ^

cc1101.h:224: error: 'SPIF' was not declared in this scope
   while (!(SPSR & _BV(SPIF)));                     // wait until SPI operation is terminated
                       ^

packages\esp8266\hardware\esp8266\2.3.0\cores\esp8266/Arduino.h:184:25: note: in definition of macro '_BV'
 #define _BV(b) (1UL << (b))
                         ^

exit status 1
'SPDR' was not declared in this scope